docs/xrpl-wallet/api-reference/usage.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Using XRPL embed SDK
+sidebar_label: Usage
+---
+
+## `login`
+
+Prompts the user to login if they are not logged in. If an OAuth verifier is not provided, a modal selector will be shown.
+
+```javascript
+await torus.login(params);
+```
+
+**Parameters**
+
+| Name             | Type                  | Description                                                                 | Mandatory |
+| ---------------- | --------------------- | --------------------------------------------------------------------------- | --------- |
+| `loginProvider?` | `LOGIN_PROVIDER_TYPE` | The login provider to use. If not provided, a modal selector will be shown. | No        |
+| `login_hint?`    | `string`              | The login hint to be passed for email_passwordless login.                   | No        |
```

Comment:
the login_hint to be passed doesn't make sense, we should change it to The login hint (user login email) to be passed for email_passwordless login.